Jenkinsfile

```diff
@@ -64,12 +64,12 @@ def rmProt(String url) {
 /**
 * Targeted scope
 */
-def TARGET_SCOPE = "@brightside"
+def TARGET_SCOPE = "@zowe"
 
 /**
 * Brightside tag to be installed
 */
-def BRIGHTSIDE_TAG_VERSION = "@next"
+def BRIGHTSIDE_TAG_VERSION = "@daily"
```

Comment:
I was thinking we could change this to say ZOWE_TAG_VERSION.
However, I wouldn't worry too much since the Jenkinsfile will be re-written in the near future to leverage the NodeJS pipeline libraries :smile:

Reply:
Addressed in this [commit](https://github.com/zowe/zowe-cli-cics-plugin/commit/1667e9f8c7ebb292f9903f3c9f4b07df4779007f)